front/app/components/AddEventToCalendarButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+// components
+import { Box, Text, Button, Error } from '@citizenlab/cl2-component-library';
+
+// intl
+import messages from './messages';
+import { useIntl } from 'utils/cl-intl';
+
+// util
+import { requestBlob } from 'utils/request';
+import { API_PATH } from 'containers/App/constants';
+import saveAs from 'file-saver';
+
+export const AddEventToCalendarButton = ({ eventId }: { eventId: string }) => {
+  const { formatMessage } = useIntl();
+  let isError = false;
```

Comment:
Let's just use `useState` for this, I don't think it will work like this

Reply:
It seems to work, but you're probably right and there is a case where it might fail :thinking: Will change it now! Thanks :)